client/errors.go

```diff
@@ -86,3 +87,19 @@ func isCodeThrottle(code string) bool {
 	_, ok := throttleCodes[code]
 	return ok
 }
+
+var (
+	requestIdRegex = regexp.MustCompile(`RequestID: [0-9a-f-]+`)
+	arnIdRegex     = regexp.MustCompile(`\sarn:aws:.+?\s`)
```

Comment:
This will only find arn's in the [commercial regions](https://miparnisariblog.wordpress.com/2020/10/26/aws-partitions/). This will not find any arns in: 

`aws-cn`
`aws-us-gov`
`aws-iso`
`aws-iso-b`

Reply:
Good catch! will fix (`aws.*?:` maybe)

Reply:
Yeah I think that would work